File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SaveLoadExpanded {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        TreeListViewState treeListViewState;

        private void Form1_Load(object sender, System.EventArgs e) {
            new DevExpress.XtraTreeList.Design.XViews(treeList1);
            treeListViewState = new TreeListViewState(treeList1);
        }

        private void btnSave_Click(object sender, System.EventArgs e) {
            treeListViewState.SaveState();
        }

        private void btnLoad_Click(object sender, System.EventArgs e) {
            treeListViewState.LoadState();
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms

Namespace SaveLoadExpanded
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private treeListViewState As TreeListViewState

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.Load
			Dim TempXViews As DevExpress.XtraTreeList.Design.XViews = New DevExpress.XtraTreeList.Design.XViews(treeList1)
			treeListViewState = New TreeListViewState(treeList1)
		End Sub

		Private Sub btnSave_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles btnSave.Click
			treeListViewState.SaveState()
		End Sub

		Private Sub btnLoad_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles btnLoad.Click
			treeListViewState.LoadState()
		End Sub
	End Class
End Namespace